accessories/ledlight.coffee

```diff
@@ -66,13 +66,48 @@ module.exports = (env) ->
           if value == @getHue()
             callback()
             return
-          hex = Please.HSV_to_HEX(h: value, s: 1, v: 1)
-          @handleVoidPromise(device.setColor(hex), callback)
+          
+          if device.hasAction("setColor")
```

Comment:
i think there is no need to check if the action exists. I do these checks just if these are methods that don't belong to the base device (base.coffee in pimatic-led-light). And as far as i know there is no implementation yet that uses the action 'setHue'. And i think we should not use methods that don't exist, because we don't know anything about their signature. Maybe setHue will needs a transformed value for hue.